### PR TITLE
Fix eslint violations on windows

### PR DESCRIPTION
**Pull request by slaweet**
### What issue have I solved?

After https://github.com/LiskHQ/lisk-hub/pull/2139 some eslint rules were broken on windows (see below)

### How have I implemented/fixed it?

- [x] Run `npm run eslint -- --fix` on windows, which added a new line to all violations. 
- [x] I manually changed most of the places to use template literals, so that the new lines are not necessary and to prevent that someone removes the new line in future without knowing.

### How has this been tested?

Run `npm run eslint` on windows

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### `npm run eslint` output on windows
```


> lisk-hub@1.19.1 eslint C:\Users\slaweet\git\lisk-hub
> eslint ./


C:\Users\slaweet\git\lisk-hub\src\components\converter\converter.js
              ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\delegates\votingHeader.js
                  ` must be placed on a new line    react/jsx-one-expression-per-line
                  ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\delegatesListView\delegateRow.js
          ` must be placed on a new line  react/jsx-one-expression-per-line
        ` must be placed on a new line    react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\secondPassphrase\firstStep.js
        ` must be placed on a new line                                                                                                                    react/jsx-one-expression-per-line
        ` must be placed on a new line                                                                                                                                                                  react/jsx-one-expression-per-line
        ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\singleTransaction\transactionDetailView\transactionVotes.js
              ` must be placed on a new line        react/jsx-one-expression-per-line
            ` must be placed on a new line          react/jsx-one-expression-per-line
                    ` must be placed on a new line  react/jsx-one-expression-per-line
              ` must be placed on a new line        react/jsx-one-expression-per-line
            ` must be placed on a new line          react/jsx-one-expression-per-line
                    ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\toolbox\flashMessage\content.test.js
      ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\toolbox\flashMessage\holder.test.js
    ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\toolbox\inputs\demo.js
      ` must be placed on a new line      react/jsx-one-expression-per-line
        ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\toolbox\pageHeader\demo.js
        ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\toolbox\tooltip\demo.js
        ` must be placed on a new line  react/jsx-one-expression-per-line
        ` must be placed on a new line  react/jsx-one-expression-per-line
        ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\transactions\transactionDetail.js
      ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\transactions\transactionRow.js
          ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\transactionSummary\index.js
              ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\voting\voteList.js
            ` must be placed on a new line              react/jsx-one-expression-per-line
          ` must be placed on a new line                react/jsx-one-expression-per-line
                        ` must be placed on a new line  react/jsx-one-expression-per-line

C:\Users\slaweet\git\lisk-hub\src\components\voting\votingSummary.js
          ` must be placed on a new line  react/jsx-one-expression-per-line



✖ 29 problems (29 errors, 0 warnings)
  29 errors and 0 warnings potentially fixable with the `--fix` option.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! lisk-hub@1.19.1 eslint: `eslint ./`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the lisk-hub@1.19.1 eslint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\slaweet\AppData\Roaming\npm-cache\_logs\2019-08-08T08_29_57_033Z-debug.log
```

### CODE REVIEW

File: src/components/delegates/votingHeader.js [hunk 1]
```diff
@@ -43,8 +43,7 @@ class VotingHeader extends React.Component {
               <span className={styles.box}>
                 <h2>
                   <span className="total-voting-number">{getTotalVotesCount(votes)}</span>
-/
-                  {maxCountOfVotes}
+                  { `/${maxCountOfVotes}`}
```
Comment (osvaldovega) [inline src/components/delegates/votingHeader.js]: Here is a extra space at the beginning

File: src/components/secondPassphrase/firstStep.js [hunk 2]
```diff
@@ -14,13 +14,19 @@ const FirstStep = ({
...
         {' '}
         <strong>required </strong>
+
+
         when
         {' '}
         <strong>confirming every transaction</strong>
         {' '}
+
+
```
Comment (osvaldovega) [inline src/components/secondPassphrase/firstStep.js]: I probably don't understand the purpose but looks like there is a lot of empty spaces here, should be like that or not?

File: src/components/toolbox/flashMessage/content.test.js [hunk 3]
```diff
@@ -42,6 +42,7 @@ describe('FlashMessage', () => {
     const children = (
       <p>
         <strong>Dummy</strong>
+
```
Comment (osvaldovega) [inline src/components/toolbox/flashMessage/content.test.js]: same here

File: src/components/toolbox/flashMessage/holder.test.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ describe('Flash Message Holder', () => {
   let wrapper;
   const DummyMessage = (
     <FlashMessage shouldShow>
+
```
Comment (osvaldovega) [inline src/components/toolbox/flashMessage/holder.test.js]: same here

**Comment by slaweet** [comment]
- [x] I updated `eslint-plugin-react@latest`. Now it seems there are no more problems on Windows, so I removed the extra empty lines. Please check again @osvaldovega

**Pull request merged**